packages/block-library/src/html/edit.js

```diff
@@ -76,10 +76,13 @@ class HTMLEdit extends Component {
 				<Disabled.Consumer>
 					{ ( isDisabled ) =>
 						isPreview || isDisabled ? (
-							<SandBox
-								html={ attributes.content }
-								styles={ styles }
-							/>
+							<>
+								<SandBox
+									html={ attributes.content }
+									styles={ styles }
+								/>
+								<div className="wp-block-html__preview-overlay"></div>
```

Comment:
Since this is an internal/editor div, it should follow our CSS class guidelines, it should be something like

`block-library-html__preview-overlay`

Reply:
fixed